docs/pt/docs/benchmarks.md

```diff
@@ -1,6 +1,6 @@
 # Comparações
 
-As comparações independentes da TechEmpower mostram as aplicações **FastAPI** rodando com Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos _frameworks_ Python mais rápidos disponíveis</a>, somente atrás dos próprios Starlette e Uvicorn (utilizados internamente pelo FastAPI). (*)
+As comparações independentes da TechEmpower mostram as aplicações **FastAPI** rodando com Uvicorn como <a href="https://www.techempower.com/benchmarks/#section=test&runid=7464e520-0dc2-473d-bd34-dbdfd7e85911&hw=ph&test=query&l=zijzen-7" class="external-link" target="_blank">um dos _frameworks_ Python mais rápidos disponíveis</a>, junto com os próprios Starlette (utilizado internamente pelo FastAPI).
```

Comment:
It would be better to have the update in the original docs as one PR and then the update for the Portuguese version as a secondary PR that can be reviewed independently, and after the first one is concluded.

Reply:
Where did I say that?

Reply:
Ah, well... I don't remember.

But since we were on the same place, I might have said "...but check with him first", pointing at Sebastian hahaha didn't I?